src/index.html

```diff
@@ -22,6 +22,363 @@
+        >
+          Buy
+        </button>
+      </article>
+      <article class="card">
+        <div class="card__image"></div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__assessment">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-cost">$2,199</span>
+        </div>
+
+        <button
+          class="card__button"
+          type="button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </article>
+      <article class="card">
+        <div class="card__image"></div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__assessment">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-cost">$2,199</span>
+        </div>
+
+        <button
+          class="card__button"
+          type="button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </article>
+      <article class="card">
+        <div class="card__image"></div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__assessment">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-cost">$2,199</span>
+        </div>
+
+        <button
+          class="card__button"
+          type="button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </article>
```

Comment:
All product cards have identical content, including the title, product code, reviews, and price. This might not be intentional and could lead to a poor user experience. Consider updating each card with unique content to accurately represent different products.